website/docs/r/cse_log_mapping.html.markdown

```diff
@@ -0,0 +1,104 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_cse_log_mapping"
+description: |-
+  Provides a CSE Log Mapping
+---
+
+# log_mapping
+Provides a CSE Log Mapping.
+
+## Example Usage
+```hcl
+resource "sumologic_cse_log_mapping" "log_mapping" {
+  name = "New Log Mapping"
+  product_guid = "003d35b3-3ba8-4e93-8776-e5810b4e243e"
+  record_type = "Audit"
+  enabled = "true"
+  relates_entities = "true"
+  skipped_values = ["skipped"]
+  fields {
+    name = "action"
+    value = "action"
+    value_type = "constant"
+    skipped_values = ["-"]
+    default_value = ""
+    format = "JSON"
+    case_insensitive = "false"
+    alternate_values = ["altValue"]
+    time_zone = "UTC"
+    split_delimiter = ","
+    split_index = "index"
+    field_join = ["and"]
+    join_delimiter = ""
+    format_parameters = ["param"]
+    lookup {
+      key = "tunnel-up"
+      value = "true"
+    }
+  }
+  structured_inputs  {
+    event_id_pattern = "vpn"
+    log_format = "JSON"
+    product = "fortinate"
+    vendor = "fortinate"
+  }
+}
+```
+
+## Argument reference
+
+The following arguments are supported:
+
+- `name` - (Required) The name of the log mapping.
+- `parent_id` - (Optional) The id of the parent log mapping.
+- `product_guid` - (Required) Product GUID.
+- `record_type` - (Required) The record type to be created. ( possible values: Audit, AuditChange, AuditFile, AuditResourceAccess, Authentication, AuthenticationPrivilegeEscalation, Canary, Email, Endpoint, EndpointModuleLoad, EndpointProcess, Network, NetworkDHCP, NetworkDNS, NetworkFlow, NetworkHTTP, NetworkProxy, Notification, NotificationVulnerability )
```

Comment:
Can we remove the extra space after the open parentheses and before the close parentheses here?